torchmdnet/models/torchmd_t.py

```diff
@@ -76,7 +76,11 @@ class TorchMD_T(nn.Module):
             (default: :obj:`None`)
         check_errors (bool, optional): Whether to check for errors in the distance module.
             (default: :obj:`True`)
-
+        additional_labels (Dict[str, Any], optional): Define the additional method to be used by the model, and the parameters to initialize it.
+            example:
+            additional_labels = {method_name1: {label_name: 'extra_arg_label', 'method_prm1': method_prm1,  'method_prm2': method_prm2}, 
+                                 method_name2: {label_name: 'extra_arg_label', 'method_prm1': method_prm1,  'method_prm2': method_prm2},
+                                 ...}
```

Comment:
T is marked as deprecated and will be removed at some point, I do not think it is worth the dev effort of adding this feature.

Reply:
ok but if someone wants do it, it's there

Reply:
That is fine, but at least reflect that it is not allowed to send it now and make the code fail if it is not None. IMO this is just adding additional unnecessary clutter to the code. 
It would be better to have a section in the documentation in the part about adding a new representation model saying "How to handle additional labels"